.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:8u171-jdk
+      - image: circleci/openjdk:9.0.4-jdk
```

Comment:
I had some errors with maven plugins, wanted to minimise the delta of this change.